modules/osdk-monitoring-prometheus-servicemonitor-creating.adoc

```diff
@@ -11,22 +11,33 @@ newly created Service.
 
 .Prerequisites
 
-- Go-based Operator generated using the Operator SDK
-- Kubernetes-based cluster with the Prometheus Operator deployed
+* Go-based Operator generated using the Operator SDK
+* Kubernetes-based cluster with the Prometheus Operator deployed
 
 .Procedure
 
-. Add the `metrics.CreateServiceMonitor()` helper function to your Operator code,
-creating only one ServiceMonitor per application and namespace:
+* Add the `metrics.CreateServiceMonitor()` helper function to your Operator code:
 +
 [source,go]
 ----
-    import "github.com/operator-framework/operator-sdk/pkg/metrics"
+import(
+    "k8s.io/api/core/v1"
+    "github.com/operator-framework/operator-sdk/pkg/metrics"
+)
+func main() {
 
-    serviceMonitors, err := metrics.CreateServiceMonitors(config *rest.Config, ns string, services []*v1.Service) <1>
+    ...
+    // Populate below with the Service(s) for which you want to create ServiceMonitors.
+    services := []*v1.Service{}
+    // Create one ServiceMonitor per application per namespace.
+    // Change the below value to name of the Namespace you want the ServiceMonitor to be created in.
+    ns := "default"
+
+    // Pass the Service(s) to the helper function, which in turn returns the array of ServiceMonitor objects.
+    serviceMonitors, err := metrics.CreateServiceMonitors(restConfig, ns, services)
```

Comment:
@LiliC Do we need to add an instruction for the `restConfig` argument?

Reply:
Yes, that makes sense! This is usually done in the main.go file, so the `restConfig` refers to https://github.com/operator-framework/operator-sdk-samples/blob/master/memcached-operator/cmd/manager/main.go#L69

Reply:
@LiliC So should I add the following lines below the "ns := "default" line?

```
// Get a config to talk to the apiserver
restConfig := config.GetConfig()
```

Reply:
@LiliC, @jianzhangbjz I added the above line and updated the PR if you want to take a look. Let me know whether that's right or not, thanks!

Reply:
@bergerhoffer Thanks! But, for me, I think we just need to add a comment for the `restConfig` argument. @LiliC What do you suggest?

Reply:
The only thing I would change is explicitly point out it's the Kubernetes apiserver config:
```go
// restConfig is used for talking to the Kubernetes apiserver
restConfig := config.GetConfig()
```

Reply:
@jianzhangbjz @LiliC Thank you both for the reviews, can you take a look at the updates and see if this looks good now?